NewForce-Cohort-4/tabloid-full-stack-team-rose
# #5 Post Details

Type of change
 New feature (non-breaking change which adds functionality)

Testing Instructions
pull down af-postdetails. run visual studio api. run npm start. log in to app with a verified user email. click on All Posts tab. Click on Post Details button. You should see the Title, Header image, Content, Publication date, and Display name for the Post. Feel free to go back to All Posts and see the details for other posts as well.

Checklist:
 My code follows the style guidelines of this project
 I have performed a self-review of my own code
 My changes generate no new warnings or errors
 I have added test instructions that prove my fix is effective or that my feature works
